Probesys/bileto
# dev: Refactor the role form with Symfony Form

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



- in the admin, create a new role
- verify it works and the role is created
- edit the role (e.g. change the permissions)
- verify your changes are saved

## Checklist
